### ISSUE TITLE
bug terga1 => k = 1 no population formed 

### ISSUE THREAD

**Issue by KAMBUMBUANGIFABIEN999** [predomics/predomicspkg#5]
The bug is thrown by the plot function in the evolve function .

```
no non-missing arguments to min; returning Infno non-missing arguments to max; returning -InfError in plot.window(...) : need finite 'xlim' values
```

Also check guides which is now deprecated, thrown by the ```digest()```function.

```
vis dans digest(obj = res_clf, plot = TRUE) :
  crossval information unavailable for 
Avis : `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.
Avis : `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.
Avis : `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.
Avis : `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.
Avis : Removed 1 rows containing missing values (geom_hline).
Avis : Removed 1 rows containing missing values (geom_hline).
```

**Comment by eprifti** [comment]
For a ter and terinter language, there can't be models with k=1 by definition. These models are thus deleted. This threw an error during the plot of the evaluation.
```
plot(trace_evolution[!is.na(trace_evolution)], ylab = "AUC score",  
           pch = 20,col = "red",type = "l",ylim = c(0,1), 
           main = paste("Best score evolution: k=",length(pop[[1]]),sep = "")) 
```

**Comment by eprifti** [comment]
Fixed in 1.0.3 version.
